packages/react-composites/src/composites/common/ParticipantContainer.tsx

```diff
@@ -98,3 +103,11 @@ export const ParticipantListWithHeading = (props: {
     </Stack>
   );
 };
+
+const paneTitleTrampoline = (paneTitle: string, totalParticipantCount?: number): string => {
+  /* @conditional-compile-remove(total-participant-count) */
+  const participantCountString = totalParticipantCount ? `(${totalParticipantCount})` : '';
+  /* @conditional-compile-remove(total-participant-count) */
+  return `${paneTitle} ${participantCountString}`;
```

Comment:
Will this work for RTL languages? (should the (`participantCountString`) be before the title for those languages

Reply:
Also, the override string is:
![image](https://github.com/Azure/communication-ui-library/assets/2684369/fc650ddc-967f-45b6-986d-bdb5c2bad869)

Given its so generic, I don't think we should add to it.

Maybe we could update our strings.json to be `"peoplePaneSubTitle": "In this call ({participantCount})",` instead?

Reply:
This was done this way for CC reasons since we cannot CC out of the localization files, will look at this again though maybe there is a better way